client.go

```diff
@@ -229,6 +229,36 @@ func (c *Client) RemoveRelation(ctx context.Context, tuples ...Tuple) error {
 	return nil
 }
 
+// ModifyRelations adds and removes the specified relation tuples in a single
+// atomic write operation. If you want to solely add relations or solely remove
+// relations, consider using the AddRelation or RemoveRelation methods instead.
+func (c *Client) ModifyRelations(ctx context.Context, addTuples, removeTuples []Tuple) error {
+	wr := openfga.NewWriteRequest()
+	wr.SetAuthorizationModelId(c.AuthModelId)
+
+	if len(addTuples) > 0 {
+		addTupleKeys := make([]openfga.TupleKey, len(addTuples))
+		for i, tuple := range addTuples {
+			addTupleKeys[i] = tuple.toOpenFGATuple()
+		}
```

Comment:
Very optional: what about a general function `func tuplesToOpenFGATupleKeys(tuples []Tuple) []openfga.TupleKey`? We have something similar in `conv.go` in `ua-contracts.`

Reply:
+1

Reply:
Thanks for catching this! I'll make the improvement :)